app/controllers/opportunities_controller.rb

```diff
@@ -29,6 +29,44 @@ def new
     @opportunity = Opportunity.new
   end
 
+  @@permitted_params = [:name, :address, :description, :registration_url, :location_name,
```

Comment:
Is this the right way to do this, @dmtroyer? I'm trying to expose the `permitted_params` to the view so we can loop over them programmatically.